src/util/psbt/map/global.rs

```diff
@@ -188,125 +188,124 @@ impl Map for PartiallySignedTransaction {
     }
 }
 
-impl PartiallySignedTransaction {
-    pub(crate) fn consensus_decode_global<D: io::Read>(d: D) -> Result<Self, encode::Error> {
```

Comment:
I will argue not to change the function into a global one (even with limited visibility).

The whole concept of associated methods not taking `self` is about namespacing - and I do not see a reason to introduce a large BLOB of consensus-critical code change just to reformat things and have a namespace as a part of a function name and not struct abstraction.

Reply:
Fair point, thanks. I've removed the patch that made this change. That also requires removing the `in path` patch also (FTR its the first time I've ever seen it either, I had to look up how to do it :)

Reply:
PSBT is not consensus code.